test/adaptive_load/scoring_function_test.cc

```diff
@@ -0,0 +1,180 @@
+#include "external/envoy/source/common/config/utility.h"
+
+#include "adaptive_load/scoring_function_impl.h"
+#include "gtest/gtest.h"
+
+namespace Nighthawk {
+
+namespace {
+
+TEST(BinaryScoringFunctionConfigFactoryTest, GeneratesEmptyConfigProto) {
```

Comment:
can you include the method you are testing in the test name. From only the name I understand the result but not what you are testing

Reply:
Done, actually renamed every test here.